webapp/src/main/java/com/box/l10n/mojito/rest/scheduledjob/ScheduledJobWS.java

```diff
@@ -35,18 +39,159 @@ public class ScheduledJobWS {
+    this.scheduledJobStatusRepository = scheduledJobStatusRepository;
+    this.scheduledJobTypeRepository = scheduledJobTypeRepository;
   }
 
   private final ResponseEntity<ScheduledJobResponse> notFoundResponse =
       createResponse(
           HttpStatus.NOT_FOUND, ScheduledJobResponse.Status.FAILURE, "Job doesn't exist", null);
 
+  @RequestMapping(method = RequestMethod.POST, value = "/api/jobs/create")
+  @ResponseStatus(HttpStatus.OK)
+  public ResponseEntity<ScheduledJobResponse> createJob(@RequestBody ScheduledJob scheduledJob) {
+    if (scheduledJob.getRepository() == null) {
+      return createResponse(
+          HttpStatus.BAD_REQUEST,
+          ScheduledJobResponse.Status.FAILURE,
+          "Repository must be provided to create a job",
+          null);
+    }
+    if (scheduledJob.getCron() == null || scheduledJob.getCron().isEmpty()) {
+      return createResponse(
+          HttpStatus.BAD_REQUEST,
+          ScheduledJobResponse.Status.FAILURE,
+          "Cron expression must be provided to create a job",
+          scheduledJob.getUuid());
+    }
+    try {
+      if (scheduledJob.getUuid() == null) {
+        scheduledJob.setUuid(UUID.randomUUID().toString());
+      }
+      if (scheduledJob.getJobStatus() == null) {
+        scheduledJob.setJobStatus(
+            scheduledJobStatusRepository.findByEnum(ScheduledJobStatus.SCHEDULED));
+      }
+      if (scheduledJob.getJobType() != null && scheduledJob.getJobType().getId() != null) {
+        scheduledJob.setJobType(
+            scheduledJobTypeRepository
+                .findById(scheduledJob.getJobType().getId())
+                .orElseThrow(
+                    () ->
+                        new ResponseStatusException(
+                            HttpStatus.NOT_FOUND,
+                            "Job type not found with id: " + scheduledJob.getJobType().getId())));
+      } else {
+        throw new ResponseStatusException(
+            HttpStatus.BAD_REQUEST, "Job type must be provided to create a job");
+      }
+
+      scheduledJobRepository.save(scheduledJob);
+      scheduledJobManager.scheduleJob(scheduledJob);
+
+      logger.info(
+          "Job '{}' for repository '{}' was created.",
+          scheduledJob.getUuid(),
+          scheduledJob.getRepository().getName());
+      return ResponseEntity.status(HttpStatus.OK)
+          .body(
+              new ScheduledJobResponse(
+                  ScheduledJobResponse.Status.SUCCESS, "Job created successfully", scheduledJob));
+    } catch (Exception e) {
+      logger.error("Error creating job", e);
+      return createResponse(
+          HttpStatus.BAD_REQUEST,
+          ScheduledJobResponse.Status.FAILURE,
+          "Error creating job: " + e.getMessage(),
+          scheduledJob.getUuid());
+    }
+  }
+
+  @RequestMapping(method = RequestMethod.PATCH, value = "/api/jobs/{id}/update")
+  @ResponseStatus(HttpStatus.OK)
+  public ResponseEntity<ScheduledJobResponse> updateJob(
```

Comment:
```suggestion
  public ScheduledJob updateJob(
```
After an update we should return the entire job back to the client